examples/gizmos/3d_gizmos.rs

```diff
@@ -18,13 +18,33 @@ fn main() {
 
 // We can create our own gizmo config group!
 #[derive(Default, Reflect, GizmoConfigGroup)]
-struct MyRoundGizmos {}
+struct MyRoundGizmos;
 
 fn setup(
     mut commands: Commands,
+    mut linegizmos: ResMut<Assets<LineGizmoAsset>>,
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
+    let mut linegizmo = LineGizmoAsset::new();
```

Comment:
Would be good to have this example showing how to despawn or move this line.

Reply:
You can move it around and despawn it like any entity, but I do think it's worth showing off how to do that
However, this example is already very busy visually. I'd like to clean it up a bit before putting more effort into it. Ideally in a separate PR.

Reply:
Yeah, there is already too much in all examples. Maybe it's worth to make a new example just for retained gizmos. But I do agree that it should be make in another PR.